### DISCUSSION TITLE
In clr-wizard-page if i will navigate to step 5 when i  will click to 2nd step i need all the above step will disable again

### DISCUSSION THREAD

**Discussion by agupgta69** [vmware-clarity/ng-clarity#348] [Q&A]
In clr-wizard-page if i will navigate to step 5 when i  will click to 2nd step i need all the above step will disable again
like 3 4 5 will be disable again

**Comment by Jinnie** [comment]
Hope this demo can help:
https://stackblitz.com/edit/pattern-wizard-reset-to-specific-page?file=src%2Fapp%2Fapp.component.ts
